apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/ServiceCollectionExtensions.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 Google LLC
+// Copyright 2024 Google LLC
```

Comment:
Please revert this change - we leave copyright years from when the file was first introduced, where possible. (It's harder for generated code, which is generated from scratch each time.)

Reply:
Will do. I haven't changed it manually. There may be some build time script or task that updates it.

Reply:
Yes, I have some minimal reformat rules on save enabled. .editorconfig is probably one of them rules sources.